app/src/main/java/com/codepath/apps/critterfinder/models/PetSearch.java

```diff
@@ -0,0 +1,134 @@
+package com.codepath.apps.critterfinder.models;
+
+import android.util.Log;
+import android.widget.ProgressBar;
+
+import com.codepath.apps.critterfinder.PetFinderHttpClient;
+import com.loopj.android.http.JsonHttpResponseHandler;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+
+import cz.msebera.android.httpclient.Header;
+
+
+
+
+
+/**
+ * Created by srichard on 3/4/16.
+ */
+public class PetSearch {
+    //define callback interface
+    public interface PetSearchCallbackInterface {
+
+        void onPetSearchSuccess(String result);
+        void onPetSearchError(String result);
+    }
+
+    String zipCode;
+    String animalType;
+    String age;
+    String size;
+    String sex;
+    public ArrayList<PetModel> petsList;
```

Comment:
I think you just mentioned this in skype as what you are going to look at next but part of this class will get replaced with the SearchFilter model we've got right?